PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/resources/META-INF/ruleConfigs.json

```diff
@@ -51,10 +51,13 @@
       "ServiceBusReceiverAsyncClient",
       "ServiceBusProcessorClient"
     ],
-    "methods_to_check": ["receiveMode","prefetchCount"],
+    "methods_to_check": [
+      "receiveMode",
+      "prefetchCount"
+    ],
     "anti_pattern_message": "A high prefetch value in PEEK_LOCK detected. We recommend a prefetch value of 0 or 1 for efficient message retrieval."
   },
-  "DetectDiscouragedAPIUsageCheck" : {
+  "DetectDiscouragedAPIUsageCheck": {
```

Comment:
Isn't the actual rule definition on L71?

Reply:
there's discouraged APIs and discouraged clients